charts/nri-bundle/Chart.yaml

```diff
@@ -68,6 +68,9 @@ dependencies:
     condition: newrelic-pixie.enabled
     version: 2.0.6
 
+  # when bumping the pixie-operator-chart, also bumping the versions of
+  # olm_crd.yaml and px.dev_viziers.yaml in 
+  # https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/kubernetes.yml
```

Comment:
Once you merge https://github.com/newrelic/open-install-library/pull/902, can you update this url to point to the exact line?

Reply:
> Once you merge [newrelic/open-install-library#902](https://github.com/newrelic/open-install-library/pull/902), can you update this url to point to the exact line?

The line may change from time to time, it is not that practical that the checklist item is updated whenever the installation recipe is updated.

Reply:
agreed, we can leave the line numbers off.